docs/antrea-network-policy.md

```diff
@@ -1776,6 +1778,9 @@ cluster-wide group.
   If set with a `podSelector`, all matching Pods from Namespaces selected by the
   `namespaceSelector` will be grouped.
 
+- **nodeSelector**: The nodes selected by the nodeSelector will be grouped. Note `hostNetwork`
+  mode must be on for the pods on the node to be included in the ClusterGroup.
```

Comment:
I feel we don't want to suggest users to use hostNetwork mode for a Pod to make it included in the ClusterGroup. It's just a fact that the hostNetwork Pod will be impacted. 
Perhaps it could just say "Note Pods in hostNetwork mode on these Nodes are also effectively included in the ClusterGroup since they share the same network namespace as the Node."

Besides, use consistent writing for K8s nouns:
s/nodes/Nodes
s/pods/Pods